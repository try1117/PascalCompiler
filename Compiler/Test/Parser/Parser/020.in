program test;
var
  a : array[1..10] of array[2..20] of integer;
  // TODO: check whether left is less then right
  b : array[-10..-100] of array [-100..100] of array[0..0] of double;
begin
end.
