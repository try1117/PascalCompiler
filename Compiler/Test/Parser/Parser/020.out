Lexical exception in position (5,14) - High range limit < low range limit
